Core/HistoManager.h

```diff
@@ -47,74 +56,34 @@ class HistoManager
   HistoManager();
   ~HistoManager();
 
-  //! call once; book (create) all trees and histograms
-  void Book();
-  //! call once; save all trees and histograms
-  void Save();
-
-  void SaveEvtPack()
-  {
-    fTree->Fill();
-  };
-
-  void Clear()
-  {
-    fEventPack->Clear();
-  };
-
+  void Book(); //! call once; book (create) all trees and histograms
```

Comment:
what happens if you call it twice :)?

Reply:
If you really want you can assure that it is called only once

Reply:
Added fbookStatus parameter to control it.